liquibase-core/src/main/java/liquibase/change/core/DropNotNullConstraintChange.java

```diff
@@ -65,6 +66,28 @@ public void setColumnDataType(String columnDataType) {
         this.columnDataType = columnDataType;
     }
 
+    /**
+     * In Oracle PL/SQL, the VALIDATE keyword defines whether a not null constraint on a column in a table
```

Comment:
Don't make the comments as oracle specific, even if that is all we support right now